static/lang/re-en.json

```diff
@@ -3661,6 +3661,7 @@
                 "Color": "Select a Color.",
                 "ConsciousMind": "Select a conscious mind.",
                 "CreatureSize": "Select a creature size.",
+                "CreatureTrait": "Ostentatious Arrival Creature Trait.",
```

Comment:
This this out of the generic prompt area

Reply:
Moved

It's now under Summoner > OstentatiousArrival >Prompt